ts/torch_handler/image_classifier.py

```diff
@@ -15,26 +14,18 @@ class ImageClassifier(VisionHandler):
     and returns the name of object in that image.
     """
 
-    topk = 5
     # These are the standard Imagenet dimensions
     # and statistics
-    image_processing = transforms.Compose([
-        transforms.Resize(256),
-        transforms.CenterCrop(224),
-        transforms.ToTensor(),
-        transforms.Normalize(mean=[0.485, 0.456, 0.406],
-                             std=[0.229, 0.224, 0.225])
-    ])
-
-    def set_max_result_classes(self, topk):
-        self.topk = topk
-
-    def get_max_result_classes(self):
-        return self.topk
+    image_processing = transforms.Compose(
```

Comment:
I think we could be more aggressive, this is fairly domain specific logic and should be provided by users, we can keep it so tests dont break but then i'd prefer if we move this handler to the test utils

Reply:
+1

Reply:
Agreed, all handler in torch_handler will only stay for bc reasons. The only  remaining important handler will be the base handler as it will integrate compose logic and provide functionality that users to not want to provide.

Reply:
my suspicion is very few people use them if at all, can't remember last time someone opened an issue with the builtin handlers - I might update the current RFC and just ask folks what they think - it should be fine for us to iron out a BC guarantee API by the time we reach 1.0